app/common/alerts.py

```diff
@@ -14,12 +13,20 @@ class AlertViaGovNotify:
 
     @staticmethod
     def send(email, reference):
-        notifications_client = NotificationsAPIClient(settings.NOTIFICATION_COMBINED_KEY)
-        notifications_client.send_email_notification(email_address=email,
-                                                     template_id=settings.NOTIFICATION_TEMPLATE_ID,
-                                                     personalisation=None,
-                                                     reference=reference)
-        logger.info('Sent secure message email notification', user_uuid=g.user.user_uuid)
+
+        notification = {
+            "emailAddress": email,
+            "reference": reference
+        }
+
+        url = urlparse.urljoin(settings.RM_NOTIFY_GATEWAY_URL, settings.NOTIFICATION_TEMPLATE_ID)
+
+        auth = (settings.SECURITY_USER_NAME, settings.SECURITY_USER_PASSWORD)
+
+        response = requests.post(url, auth=auth, timeout=settings.REQUESTS_POST_TIMEOUT, json=notification)
```

Comment:
You could just use `auth=settings.BASIC_AUTH` here as we define that object already in settings